Objects/Converters/ConverterRevit/ConverterRevitShared/RevitCommitObjectBuilder.cs

```diff
@@ -223,6 +223,19 @@ private static string GetCategoryId(Base conversionResult, Element revitElement)
     };
   }
 
```

Comment:
Yes, in the context of our discord chat the other day, this is much nicer to read, especially given where we are consuming it as single lines